docs/index.html

```diff
@@ -27,6 +27,7 @@
         '/en-gb/(.*)': '/$1',
         '/.*/_langs.md': '/_langs.md',
       },
+      fallbackLanguages: ['es'],
```

Comment:
```suggestion
```

This has already been implemented: https://github.com/qmk/qmk_firmware/blob/master/docs/index.html#L48

Reply:
just removed it